comfy_cli/cql/errors.py

```
@@ -0,0 +1,29 @@
+
+    runtime_message: str
+    details: dict[str, Any] = field(default_factory=dict)
+
+    def __post_init__(self):
+        # Initialize the Exception base with the message so str(e) works.
+        Exception.__init__(self, self.runtime_message)
+
+    def __str__(self) -> str:  # pragma: no cover - trivial
+        return self.runtime_message
+
+    def as_details(self) -> dict[str, Any]:
```

Pylint (4.0.5)

[convention] Missing function or method docstring

(C0116)